devenv/dev-dashboards/panel-near-misses-v1/near-misses-v1.json

```
@@ -0,0 +1,1031 @@
+{
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": {
+            "type": "grafana",
+            "uid": "-- Grafana --"
+          },
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
+          "name": "Annotations & Alerts",
+          "target": {
+            "limit": 100,
+            "matchAny": false,
+            "tags": [],
+            "type": "dashboard"
+          },
+          "type": "dashboard"
+        }
+      ]
+    },
+    "description": "",
+    "editable": true,
+    "fiscalYearStartMonth": 0,
+    "graphTooltip": 0,
+    "id": 2,
+    "iteration": 1668600008779,
+    "links": [],
+    "liveNow": false,
+    "panels": [
+      {
+        "datasource": {
+          "type": "simpod-json-datasource",
+          "uid": "wW6tBwHVk"
```

All the `wW6tBwHVk` values in this file need to be replaced with `json-api-near-misses`
```suggestion
          "uid": "json-api-near-misses"
```

---

have pushed this change too